### ISSUE TITLE
[Null Bug]: "Spoil Everything" with filled or conflicting data appears to break

### ISSUE THREAD

**Issue by foxsouns** [kbranch/Magpie#28]
tracker export: [Magpie-state-2023-01-05T16_39_45.536Z.json.zip](https://github.com/kbranch/Magpie/files/10357014/Magpie-state-2023-01-05T16_39_45.536Z.json.zip)
seed: `4BB73DD752DD646176788E6AF599BA4F`
settings: `LghsHTWRruEEl`
`dev.`?: yes

Reproducing:
- import tracker
- generate rom
- import rom into tracker spoiler
- click `Spoil Everything`
- done

there is evidently a null somewhere now: right clicks cease to function, and upon page reload all checks (sans one) are gone. i have evidently caused a conflict somewhere: i believe it was `Log Cave Mushroom Side`? that was fixed somewhere down the line, but some check, room or item, was incorrect as well, and the spoiler broke it. i know this because loading the spoiler on a fresh tracker state works fine: thusly, i broke it somewhere.

**Comment by foxsouns** [comment]
weird observation: upon fresh spoiler load, graveyard connector and d0 are unknown?

**Comment by foxsouns** [comment]
(unrelated: jesus fuck, this is the most long drawn-out power bracelet i have ever seen, and going through all the checks still and i havent found sword yet, ouch...)

**Comment by kbranch** [comment]
Oh, good catch! Looks like it's getting confused by trying to connect entrances that are already connected. I just pushed an update that clears the entrance mappings before spoiling.

> weird observation: upon fresh spoiler load: graveyard connector, rooster's grave, and d0 are all unknown? attempting to spoil them do not work, either.

Interesting, those are also missing from the spoiler log. I'm assuming that means they're vanilla - I also pushed an update that fills missing entrances in with that assumption.
